### DISCUSSION TITLE
Video showcase

### DISCUSSION THREAD

**Discussion by 2Jus2** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#94] [Show and tell]
Click the [link](https://youtu.be/vGb8e1YyzNs?si=935j2zubev3Pc6ZE) to check our project video shocase for this sem!!!!

**Comment by SeanLi755** [comment]
LGTM！
😇

**Comment by ZIYANGSONG2003** [comment]
LGTM
